cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,27 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+describe('Site page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
+  cy.get('h1').should('contain.text', 'Sign In');
+
+  cy.contains('Sign in').click();
+
+  cy.get('h1').should('contain.text', 'Sign In');
+
+  it('should fill in email and password fields with valid data', () => {
+    cy.get('[placeholder=Email]')
+      .type('oleksadowbush@gmail.com');
+
+    cy.get('[placeholder=Password]')
+      .type('Oleksa19*');
+
+    it('should click on [Sign in] button ', () => {
```

Comment:
Nested 'it' blocks are not a good practice in Cypress, the test case 'should click on [Sign in] button' should be separate or merged with another test case.